### ISSUE TITLE
Found unsolvable puzzle

### ISSUE THREAD

**Issue by OmegaGoo** [sigma144/witness-randomizer#54]
This puzzle has no solution. I marked the symbols due to the blocking filter; I have verified that these are the symbols.

Seed: Expert 1

![Screenshot 2022-02-09 164733](https://user-images.githubusercontent.com/14987288/153303636-a43c671b-84c6-45e4-b214-4b453a43e67e.png)

**Comment by sigma144** [comment]
Here is a solution I found for this puzzle. I tested this solution in-game and it seems to work.

<details>
  <summary>Click to see solution</summary>
  
![Capture](https://user-images.githubusercontent.com/3487299/153682643-143c1c35-0659-43dd-b3f4-7c2c012cb7f0.PNG)

</details>

The provided solver can be a bit glitchy and will sometimes say there is no solution when there is. This doesn't affect the randomizer though since it uses separate (and more reliable) code to generate puzzles. Hope this helps! 👍
